treeherder/model/models.py

```diff
@@ -1478,6 +1478,10 @@ class TextLogErrorMetadata(models.Model):
     class Meta:
         db_table = "text_log_error_metadata"
 
+    def __str__(self):
```

Comment:
Does this model now need a `@python_2_unicode_compatible`?

Reply:
Good catch, fixed and rebased.